evm/src/main/java/org/hyperledger/besu/evm/code/CodeV1.java

```diff
@@ -135,7 +135,7 @@ public class CodeV1 implements Code {
     VALID, // 0x59 - MSIZE
     VALID, // 0x5a - GAS
     VALID, // 0x5b - NOOOP (née JUMPDEST)
-    VALID, // 0X5c - RJUMP
+    VALID_AND_TERMINAL, // 0X5c - RJUMP
```

Comment:
Is this VALID_AND_TERMINAL to support the case where its immediate is truncated?

Reply:
No, it's to support the case where a complete RJUMP is the final instruction of a code section.  It doesn't terminate execution but it can terminate the code block when reading linearly.